tgui/packages/tgui/interfaces/Fabrication/SearchBar.tsx

```diff
@@ -26,6 +27,9 @@ export type SearchBarProps = {
 export const SearchBar = (props: SearchBarProps, context) => {
   const { searchText, onSearchTextChanged, hint } = props;
 
+  // Only actually update the search query every 200 milliseconds at most.
+  const db = debounce(onSearchTextChanged, 200);
```

Comment:
Not sure if that even works, you create a new debounced version of this function on every render.